docs/docs/widgets/button-group.md

```diff
@@ -18,20 +18,18 @@ The **Button Group** component is used to group a series of buttons together in
 
 | <div style={{ width:"100px"}}> Properties </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:----------- |:----------- |:-------------- |
-| label | Sets the title of the button-group. | Any **String** value: `Select the options` or `{{queries.queryname.data.text}}` |
-| values | Sets the values of the Button Group items. | **Array** of strings and numbers: `{{[1,2,3]}}` |
-| Labels | Sets the labels of the Button Group items. | **Array** of strings and numbers: `{{['A','B','C']}}` |
+| label | Sets the title of the button-group. | Any **String** value: `Select the options` or `{{queries.queryname.data.text}}`. |
+| values | Sets the values of the Button Group items. | **Array** of strings and numbers: `{{[1,2,3]}}`. |
+| Labels | Sets the labels of the Button Group items. | **Array** of strings and numbers: `{{['A','B','C']}}`. |
 | Default selected | Sets the initial selected values. | **Array** of strings and numbers: `{{[1]}}` will select the first button by default. |
-| Enable multiple selection | Toggle on or off to enable multiple selection. | **Boolean** value: `{{true}}` or `{{false}}` |
+| Enable multiple selection | Toggle on or off to enable multiple selection. | **Boolean** value: `{{true}}` or `{{false}}`. |
 
 </div>
 
 <div style={{paddingTop:'24px', paddingBottom:'24px'}}>
 
 ## Events
 
-Events are actions that can be triggered programmatically when the user interacts with the component. Click on the component handle to open its properties on the right. Go to the **Events** accordion and click on **+ Add handler**.
-
 | <div style={{ width:"100px"}}> Events </div> | <div style={{ width:"100px"}}> Description </div> |
 |:----------- |:----------- |
 | On click | This event is triggered when the user clicks on the button in the Button Group. |
```

Comment:
Hey @HareeshDhruva, as mentioned in the issue, you have to update the content of the description column here. ([reference](https://docs.tooljet.com/docs/widgets/text-input/#events) (Please just check the sentence structure from the given reference))

Reply:
I noticed that clicking the "reference" redirects to the text input components section. To ensure consistency, I would like to follow the sentence structure used there for the event descriptions in our documentation.